docs/authentication/saml/azure.mdx

```diff
@@ -101,16 +107,35 @@ description: Learn how to allow users to sign into your Clerk app with their Mic
 
   The only Microsoft claim that is necessary to map is the email address claim. This is the email address that your users will use to log in to your application.
 
+  1. Navigate back to the **Microsoft Azure portal**.
   1. On the **Set up Single Sign-On with SAML** page, find the **Attributes & Claims** section.
   1. Select the **Edit** button.
-  1. Select the email address claim (`http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress`) to edit the field.
-  1. Next to **Source attribute**, select the dropdown and choose `user.userprincipalname`.
+  1. To edit the email claim, select the claim with the value of **`user.email`**. You will be redirected to the **Manage claim** page.
+  1. Next to **Source attribute**, search and select the dropdown and choose **`user.userprincipalname`**.
```

Comment:
search and select? what did you mean here
also we don't need to add bold around a code snippet